Docker/docker-compose.multi-app.yaml

```
@@ -0,0 +1,60 @@
+    volumes:
+      - ../vo_new:/app
+      - /app/node_modules
+    networks:
+      - dalgo-network
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    command: npm run dev
+
+networks:
+  dalgo-network:
+    driver: bridge 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)